lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -170,20 +170,23 @@ def make_request(request, params = nil)
         read_response
       end
 
-      sig { params(request: String, params: T.nilable(T::Hash[Symbol, T.untyped])).void }
+      # Messages expect a response back
```

Comment:
That's not correct. Only requests expect a response back (the method above this one), messages are equivalent to notifications.

Reply:
No, I think `send_message` still makes sense. But the comment doesn't belong here though.

Sending a notification -> send_message (aliased to send_notification to match the spec)
Sending a request -> send_message + read_response